.github/workflows/causal_lm_cpp.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Install OpenVINO
         run: |
           mkdir ./ov/
-          curl https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/2023.3.0-13739-294cc6668c4/l_openvino_toolkit_ubuntu20_2023.3.0.dev20231219_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
+          curl https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/2024.0.0-14049-0dc2dc5f602/l_openvino_toolkit_ubuntu20_2024.0.0.dev20240118_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
```

Comment:
is it supposed to return back to 2023.3 once official release is out?

Reply:
I don't have objections to set 23.3 back. The problem is that there's no longer nightly 23.3

Reply:
- we are on releases/2023/3 branch, so it's expected to test against corresponding release
- current nightly release will be removed in future as well and you will have to update it one more time.